hw/ip/otbn/dv/rig/rig/gens/known_wdr.py

```diff
@@ -0,0 +1,133 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import random
+from typing import List, Optional
+
+from shared.operand import EnumOperandType, ImmOperandType, RegOperandType
+from shared.insn_yaml import InsnsFile
+
+from ..config import Config
+from ..model import Model
+from ..program import ProgInsn, Program
+
+from ..snippet import ProgSnippet
+from ..snippet_gen import GenCont, GenRet, SnippetGen
+
+
+class KnownWDR(SnippetGen):
+    '''A snippet generator that generates known values (all zeros or all ones
+    for now) for WDRs.
+
+    '''
+
+    def __init__(self, cfg: Config, insns_file: InsnsFile) -> None:
+        super().__init__()
+
+        if 'bn.xor' not in insns_file.mnemonic_to_insn:
+            raise RuntimeError('BN.XOR instruction not in instructions file')
+
+        if 'bn.not' not in insns_file.mnemonic_to_insn:
+            raise RuntimeError('BN.NOT instruction not in instructions file')
```

Comment:
You don't need this any more because `_get_named_insn` has that error handling built in.